qualtran/drawing/_show_funcs.py

```diff
@@ -84,3 +86,12 @@ def show_flame_graph(*bloqs: 'Bloq', **kwargs):
     """Display hiearchical decomposition and T-complexity costs as a Flame Graph."""
     svg_data = get_flame_graph_svg_data(*bloqs, **kwargs)
     IPython.display.display(IPython.display.SVG(svg_data))
+
+
+def show_bloq_via_qpic(bloq: 'Bloq', width: int = 1000, height: int = 400):
```

Comment:
this should likely just be a different argument for `type` in `show_bloq`

Reply:
@tanujkhattar ^

Reply:
I consider this to still be pretty experimental so wanted to keep it separate, but I can add it to `show_bloq` as well if you think it's fine.

Reply:
well let's not make it the default yet, but the "spirit" of the show functions is to provide the simplest possible entry point to getting something to show up in a notebook. I think the `type` parameter is perfect for this case